cmd/bucky/common_sync.go

```diff
@@ -330,14 +349,18 @@ func (ms *metricSyncer) sync(jobc chan *syncJob, srcThrottling map[string]chan s
 					} else if errors.Is(err, errCantReadMetric) {
 						atomic.AddInt64(&ms.stat.copyError, 1)
 						atomic.AddInt64(&ms.stat.nodes[src].copyError, 1)
+						ms.metriclog("Src", job.oldName, src)
```

Comment:
Since "Src" & "Dst" are used in multiple lines, maybe it makes sense to get a bool as the argument or create a couple of consts to give as arguments e.g. SyncErrSrc, SyncErrDst.

Personally, I would go with the first option.

Reply:
bool looks ok, done